test/verify/machineslib.py

```diff
@@ -167,7 +168,7 @@ def readFile(name):
 
 
 @skipImage("Atomic cannot run virtual machines", "fedora-atomic", "rhel-atomic", "continuous-atomic")
-class TestMachines(MachineCase):
+class TestMachines(NetworkCase, MachineCase):
```

Comment:
Please don't do this, it's confusing. NetworkCase already inherits from MachineCase.